tests/PCTCompile/test79/build.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<project name="PCTCompile-test79">
+  <taskdef resource="PCT.properties" />
+
+  <mkdir dir="build" />
+
+  <target name="test1">
+    <PCTCompile destDir="build" dlcHome="${DLC}" relativePaths="true" failOnError="false" outputType="json">
+      <fileset dir="src" includes="**/test1.p" />
+      <propath location="src" />
+      <Profiler enabled="${PROFILER}" coverage="true" outputDir="profiler" />
+    </PCTCompile>
+  </target>
+
+   <target name="test2">
+    <PCTCompile destDir="build" dlcHome="${DLC}" relativePaths="true" failOnError="false" outputType="JSON">
+      <fileset dir="src" includes="**/test2.p" />
+      <propath location="src" />
+      <Profiler enabled="${PROFILER}" coverage="true" outputDir="profiler" />
+    </PCTCompile>
+  </target>
+
+  <target name="test3">
+    <PCTCompile destDir="build" dlcHome="${DLC}" relativePaths="true" failOnError="false" outputType="json">
+      <fileset dir="src" includes="**/test3.p" />
+      <propath location="src" />
+      <Profiler enabled="${PROFILER}" coverage="true" outputDir="profiler" />
+    </PCTCompile>
+  </target>
+
+  <target name="test4">
+    <PCTCompile destDir="build" dlcHome="${DLC}" relativePaths="true" failOnError="false" outputType="json">
+      <fileset dir="." includes="rssw/pct/TestClass2.cls" />
```

Comment:
Test with files in the 'src' directory, I think this test in '.' is just the result of the copy/paste of an existing test case (which was testing relativePaths='true'). Just to keep the tests clean...